pom.xml

```diff
@@ -38,7 +38,7 @@
   <properties>
     <tycho.version>1.1.0</tycho.version>
     <tycho-extras.version>1.0.0</tycho-extras.version>
-    <product.version.qualifier.postfix/>  <!-- 0-length string by default -->
+    <product.version.qualifier.suffix/>  <!-- 0-length string by default -->
```

Comment:
Drop `product.` since this is applying to all artifacts?

Reply:
Will fix #3038 soon, so `product` will be OK?

Reply:
Sure

Reply:
Decided to fix #3038 here.
- Removed `<format>` from the top-level `pom.xml`.
- Added `<format>` to `gcp-repo/metadata.product`.
- Added `<format>` to `features/com.google.cloud.tools.eclipse.suite.feature/pom.xml`.

It works well. Only the feature version and the product branding version have the suffix when set.